processes/ingest/hathi_trust.py

```diff
@@ -72,7 +72,15 @@ def importFromHathiTrustDataFile(self, fullDump=False):
         )
 
         for hathiFile in fileJSON:
+            print(hathiFile)
             if hathiFile['full'] == fullDump:
+                if hathiFile['full'] == False:
```

Comment:
Hmm - I think we could simplify this boolean logic a bit. Basically, we are saying if we are not doing a full dump and the file is not full then ingest records using a start date time.  If we are doing a full dump and the hathiFile is full, then import the whole file. 

This reads nested if statement reads if the hathiFile full field is equal to our fullDump param and the hathiFile full field is false, then ingest using start date time. 

I also recommending using .get('full') just to guard against missing fields.

Reply:
These are all good suggestions Kyle. Thank you and I'll implement them in my next commit.